samples/cellular/nrf_cloud_mqtt_multi_service/overlay_nrf7002ek_wifi_no_lte.conf

```diff
@@ -0,0 +1,125 @@
+# Note that this overlay is not compatible with nrf91! Use something like the nrf53 as the host
+# microcontroller.
+
+## Disable Modem/LTE/nrf91-specific features
+CONFIG_NRF_MODEM_LIB=n
+CONFIG_MODEM_INFO=n
+CONFIG_MODEM_INFO_ADD_NETWORK=n
+CONFIG_NRF_MODEM_LIB=n
+CONFIG_AT_HOST_LIBRARY=n
+CONFIG_LTE_LINK_CONTROL=n
+
+## Disable offloaded sockets (we are going to switch to the native networking stack)
+CONFIG_NET_SOCKETS_OFFLOAD=n
+
+## Disable cloud features reliant on or specific to nrf91 modem features
+CONFIG_NRF_CLOUD_AGPS=n
+CONFIG_NRF_CLOUD_PGPS=n
+CONFIG_NRF_CLOUD_PGPS_REQUEST_UPON_INIT=n
+CONFIG_LOCATION_METHOD_GNSS=n
+CONFIG_LOCATION_METHOD_CELLULAR=n
+
+## Disable FOTA to conserve memory
+CONFIG_NRF_CLOUD_FOTA=n
+CONFIG_BOOTLOADER_MCUBOOT=n
+CONFIG_IMG_MANAGER=n
+CONFIG_MCUBOOT_IMG_MANAGER=n
+
+## Disable LTE-event-driven date_time updates (the date_time library will still periodically
+## refresh its timestamp, this setting only controls whether LTE-event-driven date_time updates
+## are enabled)
+CONFIG_DATE_TIME_AUTO_UPDATE=n
+
+## Disable LED patterns since enabling WiFi shield takes control of two LEDs
+CONFIG_LED_INDICATION_DISABLED=y
+
+## Disable LTE conn_mgr bindings
+CONFIG_LTE_CONNECTIVITY=n
+
+## These had to be disabled for LTE with conn_mgr, but there is no reason to disable them for
+## wifi with conn_mgr, and doing so inadverdently disables DNS resolution, so re-enable them.
+CONFIG_NET_IPV6_NBR_CACHE=y
+CONFIG_NET_IPV6_MLD=y
+CONFIG_DNS_RESOLVER=y
+
+## Enable Wi-Fi drivers, (and the native NET stack so that the location library can access them)
+CONFIG_WIFI=y
+CONFIG_WIFI_NRF700X=y
+
+## Enable Wi-Fi conn_mgr bindings
+CONFIG_L2_WIFI_CONNECTIVITY=y
+CONFIG_NET_CONNECTION_MANAGER_STACK_SIZE=4096
+
+## Enable WIFI_MGMT_EXT and WIFI_CREDENTIALS for simplified Wi-Fi connection setup
+CONFIG_WIFI_CREDENTIALS=y
+CONFIG_WIFI_MGMT_EXT=y
+CONFIG_WIFI_CREDENTIALS_BACKEND_SETTINGS=y
+
+## Use compile-time client ID for nRF Cloud
+CONFIG_NRF_CLOUD_CLIENT_ID_SRC_COMPILE_TIME=y
+
+## TODO: Needed...?
```

Comment:
Oops

Reply:
I need to take a closer look at this, I will do so separately from the review process